src/api/is-device-supported.js

```diff
@@ -0,0 +1,15 @@
+import getDeviceSize from '../utils/get-device-size';
+
+
+/**
+ * Checks if the device is supported as the target for the `resize` API method.
+ * @function
+ * @async
+ * @name isDeviceSupportedAsResizeTarget
+ * @param {string} deviceName - Specifies the name of the target device. You can use the values specified in the Device Name column of [this table]{@link http://viewportsizes.com/}.
+ */
```

Comment:
We can prepare json as a part of build task: get data using `viewport-list` package and then store it in our json file